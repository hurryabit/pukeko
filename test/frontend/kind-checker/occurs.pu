data Typ f = Con (f f)
