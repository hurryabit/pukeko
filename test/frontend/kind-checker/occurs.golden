test/frontend/kind-checker/occurs.pu:2:5: occurs check
